### ISSUE TITLE
OTP login doesn't work with HA config on second node

### ISSUE THREAD

**Issue by linuxmail** [opnsense/core#2356]
hi,

I configured successful the OTP plugin and it works on the first / primary node, but fails on the secondary. The plain fallback configuration (aka password only) still works.

Which log (lines) should I submit ?

cu denny

**Comment by linuxmail** [comment]
Forgotten to say: the tester on the second node works.

**Comment by fichtner** [comment]
Can you reload "Users and Groups" from Services: Diagnostics on the secondary note? Also, you are talking about TOTP for local accounts?

Last but not least, make sure that System: Access: Settings has the same primary auth type.


Cheers,
Franco

**Comment by linuxmail** [comment]
hi,

seems to be fixed in 18.1.6. I added direct access to the internet because proxy worked only for packages, but not system upgrade (via F12).
OTP works now on both systems  :-)

**Comment by linuxmail** [comment]
hi,

**Comment by linuxmail** [comment]
Hi,

sorry, I have reopen the issue. After upgrading to the latest a version 18.1.x, OTP is not longer working. Nor on the master, nor on the slave. The test works (OTP + password) but on the login, it fails. It there any log I can read ?

**Comment by fichtner** [comment]
Are you on 18.1.x or 18.7.b ?

**Comment by linuxmail** [comment]
Version 18.1.8

**Comment by fichtner** [comment]
Nothing changed here then. Make sure NTP is in sync, and TOTP is selected as the authentication method on both devices.

**Comment by fichtner** [comment]
Not really, it will only tell you if the login is correct or not. Are you using Hyper-V by any chance?

**Comment by linuxmail** [comment]
have to close it again ... maybe I try to out later, why it won't work (sometimes).
